app/backend/templates/unseen_messages.md

```diff
@@ -0,0 +1,36 @@
+---
+subject: "You have {{ total_unseen_message_count }} unseen messages in {{ unseen_messages|length }} chats on Couchers.org!"
+---
+
+{% from "macros.html" import button %}
+
+Hi {{ escape(user.name) }}!
+
+You've unseen messages on Couchers.org, here's the latest:
+
+{% for group_chat, latest_message, count in unseen_messages[:5] %}
+{% if group_chat.title %}
+"{{ group_chat.title }}": {{ latest_message.author.name }} wrote on {{ latest_message.time }}: "{{ latest_message.text }}" ({{ count - 1 }} more messages)
+{% else %}
+Private message from {{ latest_message.author.name }} on {{ latest_message.time }}: "{{ latest_message.text }}" ({{ count - 1 }} more messages)
+{% endif %}
+
+{% endfor %}
+
+{% if unseen_messages|length > 5 %}
+... and {{ total_unseen_message_count - 5 }} more messages in {{ unseen_messages|length - 5}} other chats!
+{% endif %}
+
+Check it out here:
+
+{% if html %}
```

Comment:
So much for writing one markdown to generate both text and html, haha!

Reply:
At least they're next to each other :)

Or we could change `button` to just emit `<{{ link }}>` in non-HTML mode

Reply:
The way to do links in markdown is simply
```
[Friend Requests]({{ group_chats_link }})
```